book/makefile

```diff
@@ -109,16 +110,16 @@ book_serif/book.pdf: $(src_all)
 book_sans_serif/book_sans_serif.pdf: $(src_all)
 	$(LATEX) -output-directory=book_sans_serif book_sans_serif.tex
 
-.PHONY: copy_ebook_files copy_ebook_files_cl
+.PHONY: copy_ebook_files copy_ebook_files_low_res
 
 epub/%.epub: %.tex $(ebook_src) cover/cover-page.xbb
 	$(EBOOK) $<
 
 copy_ebook_files: build_ebook
 	$(RSYNC) epub_build/book-epub/ bw-book-epub/
 
-copy_ebook_files_cl: build_ebook
-	$(RSYNC) epub_build/book-epub/ low-res-book-epub/
+copy_ebook_files_low_res: build_ebook
+	$(RSYNC_INC_PNG) epub_build/book-epub/ low-res-book-epub/
```

Comment:
Option one you add -- exclude "*.png" to $(RSYNC) command for B&W this way we only have a single command...